creator/studies/migrations/0025_add_organizations.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.2.20 on 2021-04-29 01:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def set_default_org(apps, schema_editor):
+    """
+    Create a default organization and put all studies and users under it.
+    """
+    Organization = apps.get_model("organizations", "Organization")
+    User = apps.get_model("creator", "User")
+    Study = apps.get_model("studies", "Study")
+    org = Organization(
+        id="300d175a-4904-44e5-878b-e8ead565a082", name="Default Organization"
+    )
+    org.save()
+    org = Organization.objects.filter(name="Default Organization").first()
+
+    for user in User.objects.all().iterator():
+        user.organizations.add(org)
+        user.save()
+
+    for study in Study.objects.all().iterator():
```

Comment:
Same comment as before
```suggestion
    for study in Study.objects.only("organization").all().iterator():
```